pkg/hex/hex.go

```diff
@@ -0,0 +1,59 @@
+package hex
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"strings"
+)
+
+var EncodeToString = hex.EncodeToString
```

Comment:
Do we need these vars if we still use `hex...` functions below?

Reply:
not needed, just aliased them so users of this package don't need to import 2x hex packages.